styles/widgets/common/fileManager.less

```diff
@@ -0,0 +1,164 @@
+.dx-filemanager {
+    display: flex;
+    flex-direction: column;
+    
+    .dx-filemanager-toolbar {
+        display: flex;
+    }
+
+    .dx-filemanager-container {
+        display: flex;
+        display: -webkit-flex;
+        height: 100%;
+    }
+
+    .dx-filemanager-dirs-tree {
+        min-width: 20%;
+    }
+
+    .dx-filemanager-view-separator {
+        min-width: 2px;
+        background-color: #dddddd;
```

Comment:
Design for other themes (color themes) will be added later?

Reply:
Yes, it will be later.